pyccl/core.py

```diff
@@ -914,7 +914,7 @@ def compute_nonlin_power(self):
 
         if mps == "camb" and self._has_pk_nl:
             # Already computed
-            return
+            return self._pk_nl['delta_matter:delta_matter']
```

Comment:
This function isn't meant to return anything.

Reply:
Fixed in merging.